### PR TITLE
Add test feature 1

### PR DESCRIPTION
**Pull request by memsprop**

### CODE REVIEW

File: TEST-FEATUR-1.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+STEP1 Testing
+STEP2
```
Comment (memsprop) [inline TEST-FEATUR-1.py:2]: missing corner case
